featuretools/utils/entity_utils.py

```diff
@@ -126,6 +127,11 @@ def convert_all_variable_data(df, variable_types):
                                        new_type=desired_type,
                                        **type_args)
 
+        # Fill in any single `NaN` values in LatLong variables with a tuple
+        if issubclass(desired_type, vtypes.LatLong) and isinstance(df[var_id], pd.Series) and df[var_id].hasnans:
+            df[var_id] = replace_latlong_nan(df[var_id])
+            warnings.warn("All single `NaN` values in column `{}` have been replaced with `(NaN, NaN)`".format(var_id))
```

Comment:
I think we could work in that this conversion is happening due to the column being LatLong type

Reply:
Updated message.